README.md

```diff
@@ -1,2 +1,47 @@
 # Dmitry & Ryan Lab-7
-Networking + CAN Bus
\ No newline at end of file
+Networking + CAN Bus
+
+-Activity 0 : Sending and Recieving Messages
+By sending a single message periodically establishes the basic functionality
+of the CAN bus system with a recieve function to indicate message is received.
+
+Observed: Single message is sent to CAN bus periodically. If Red LED is on;
+          the message is received. If Red LED is off, message is not received.
+          Red LED indicates receive callback for no filter is utilized.
+
+-Activity 1
+Creating system call to send 2 messages to the CAN bus and establishing 
+receive call for filtering incoming message to match a specific filter.
+
+Observed: Two messages are sent to CAN bus periodically. If Orange LED is on;
+          the message is received with match to the filter. If Orange LED is 
+          off, message received does not match the filter.
+          Orange LED indicates receive callback for single filter 1 is utilized.
+
+-ACtivity 2
+Sending two messages on the bus line one with a high priority and one with a 
+low priority using 2 stm32 MCU boards both running the same code.
+
+Observed: When 2 messages are sent to the bus line using two microcontrollers
```

Comment:
Love the detailed explanation of your observations, It shows you are paying attention to what is happening to help others replicate behavior.